starter-code/js/index.js

```diff
@@ -1,52 +1,201 @@
-function deleteItem(e){
+function initVariables() {
 
-}
+  // default quantity : 0
+  document.querySelectorAll(".product input").forEach(x=>{
+    x.innerHTML = 0;
+  })
+
+  // unitary prices have two digits :
+  document.querySelectorAll(".product price-one-unit").forEach(x=>{
+    x.innerHTML = Number(x.innerHTML).toFixed(2);
+  })
+
+    // unitary prices have two digits :
+  document.querySelectorAll(".total-price-div span").forEach(x=>{
+    x.innerHTML = Number(x.innerHTML).toFixed(2);
+  })
+
+  getTotalPrice();
 
-function getPriceByProduct(itemNode){
 
 }
 
-function updatePriceByProduct(productPrice, index){
+function deleteItem(e){
+  e.currentTarget.parentNode.parentNode.remove();
+  getTotalPrice();
 
 }
 
 function getTotalPrice() {
 
-}
+  var bigTotal = 0;
+
+  // 1. Select product divs
+  var productDivs = document.querySelectorAll(".products .product");
+
+  // 2. For each product div
+  productDivs.forEach(function(productDiv){
+
+    // 3. get unitary price
+    var unitaryPrice = Number(productDiv.querySelector(".price-one-unit").innerHTML);
+
+    // 4. get quantity
+    var quantity = Number(productDiv.querySelector("input").value);
 
-function createQuantityInput(){
+    // 5. calculate the total price
+    var totalPrice = (unitaryPrice * quantity).toFixed(2);
+
+    // 6. assign total price to total-price span
+    productDiv.querySelector(".total-price").innerHTML = totalPrice;
+
+    // 7. update big total
+    bigTotal += Number(totalPrice);
+
+  });
+
+  // 8. render big total
+  document.querySelector(".total-price-div span").innerHTML = Number(bigTotal).toFixed(2);
 
 }
 
 function createDeleteButton(){
+  // <span><button class="btn btn-delete">Delete</button></span>
+
+  var out = document.createElement('span');
 
+  var button = document.createElement("button");
+  button.classList.add("btn");
+  button.classList.add("btn-delete");
```

Comment:
I am the one who told you to do it this way, just for information I just realised that there is also a simpler way to add several classes at once  : 
```
button.className = "btn btn-delete"
```